lib/nimble_totp.ex

```diff
@@ -124,7 +149,7 @@ defmodule NimbleTOTP do
 
   ## Options
 
-    * :time - The time in unix format to be used. Default is `System.os_time(:second)`
+    * :time - The time to be used. Default is `System.os_time(:second)`
```

Comment:
Do we still want to list all possible types? I chose to add specs so they would document this partially, but if we still want them I can add them back?

Reply:
Let's document them, yeah.  :)